db/migrate/20990101999999_sample_migration.foreman_plugin_template.rb

```diff
@@ -0,0 +1,2 @@
+class SampleMigration < ActiveRecord::Migration
```

Comment:
Don't add migrations to this template, this does nothing, and will not be removed by authors.